silta-cluster/templates/deployment-remover.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-deployment-remover
+  annotations:
+    getambassador.io/config: |
+      apiVersion: ambassador/v0
+      kind:  Mapping
+      name:  {{ .Release.Name }}-deployment-remover
+      host: webhooks.{{ .Values.clusterDomain }}
+      prefix: /
+      service: {{ .Release.Name }}-deployment-remover.{{ .Release.Namespace }}.svc.cluster.local:80
+      timeout_ms: 30000
+    domain: webhooks.{{ .Values.clusterDomain }}  
+spec:
+  type: NodePort
+  externalTrafficPolicy: Local
+  ports:
+    - port: 80
+  selector:
+    name: {{ .Release.Name }}-deployment-remover
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-deployment-remover
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        name: {{ .Release.Name }}-deployment-remover
+    spec:
+      initContainers:
+      - name: wait-redis
+        image: busybox
+        command: ['sh', '-c', 'until echo -e "AUTH $REDIS_PASSWORD\nPING" | nc $REDIS_HOST 6379; do echo waiting for redis; sleep 2; done;']
```

Comment:
`nc $REDIS_HOST 6379`  -> `nc "$REDIS_HOST" 6379`
Always quote shell variables, unless you have a good reason not to.

I also suggest to consider to use expect for such use-cases.

Reply:
Good spot! Change pushed for PR and also to the originating repo, https://github.com/wunderio/silta-cluster

Reply:
Can't use `expect` in this case though, because it's usually not in the default tools like the `busybox` image we're using.